static/styles/popovers.css

```diff
@@ -205,21 +205,20 @@ ul {
     }
 }
 
-.actions_popover_wrapper {
-    .arrow {
-        position: absolute;
-    }
+.actions_popover {
+    /* Override default padding of tippyjs */
+    margin: 0 -9px;
 
-    @media (width < $xl_min) {
-        width: max-content;
+    /* Override bootstrap defaults */
+    .nav-list > li > a {
+        padding: 3px 15px;
     }
```

Comment:
Thanks for flagging these. I imagine we'll want to stop using `nav-list` altogether for popovers in favor of our own CSS that doesn't depend on bootstrap at some point, but that's probably best saved for a separate PR.